docs/media-airplay-button.md

```diff
@@ -0,0 +1,28 @@
+# `<media-airplay-button/>`
+
+Button to jump back 30 seconds in the media.
```

Comment:
Description needs updating (I should have  pointed to it specifically before, sorry)

Reply:
booooo move fast break docs?